src/bin/helpers.js

```diff
@@ -1,20 +1,51 @@
+const fs = require("fs")
+const merge = require("lodash.merge")
+const framework = require("witnet-solidity-bridge")
+
+const commas = (number) => {
+  const parts = number.toString().split(".")
+  const result = parts.length <= 1
+    ? `${parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ",")}`
+    : `${parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ",")}.${parts[1]}`
+  return result
+}
+
+const blue = (str) => `\x1b[34m${str}\x1b[0m`
 const cyan = (str) => `\x1b[36m${str}\x1b[0m`
+const gray = (str) => `\x1b[90m${str}\x1b[0m`
 const green = (str) => `\x1b[32m${str}\x1b[0m`
+const red = (str) => `\x1b[31m${str}\x1b[0m`
 const yellow = (str) => `\x1b[33m${str}\x1b[0m`
 const white = (str) => `\x1b[98m${str}\x1b[0m`
+const lblue = (str) => `\x1b[1;94m${str}\x1b[0m`
 const lcyan = (str) => `\x1b[1;96m${str}\x1b[0m`
+const lgreen = (str) => `\x1b[1;92m${str}\x1b[0m`
 const lwhite = (str) => `\x1b[1;98m${str}\x1b[0m`
+const mblue = (str) => `\x1b[94m${str}\x1b[0m`
 const mcyan = (str) => `\x1b[96m${str}\x1b[0m`
+const mgreen = (str) => `\x1b[92m${str}\x1b[0m`
+const mred = (str) => `\x1b[91m${str}\x1b[0m`
+const myellow = (str) => `\x1b[93m${str}\x1b[0m`
 
-module.exports = {
-    colors: {
-        cyan, green, yellow, white,
-        lcyan, lwhite,
-        mcyan, 
-    },
-    deleteExtraFlags, extractFlagsFromArgs, extractOptionsFromArgs,
-    flattenObject, orderKeys,
-    showVersion, traceHeader, traceWitnetAddresses,
+const WITNET_SDK_RADON_ASSETS_PATH = process.env.WITNET_SDK_RADON_ASSETS_PATH || "../../../../witnet/assets"
+const isModuleInitialized = fs.existsSync("./witnet/assets/index.js")
+
+function readWitnetJsonFiles(...filenames) {
+    return Object.fromEntries(filenames.map(filename => [
+        filename,
+        fs.existsSync(`${WITNET_SDK_RADON_ASSETS_PATH}/../${filename}.json`) 
+            ? require(`${WITNET_SDK_RADON_ASSETS_PATH}/../${filename}.json`)
+            : {}
+    ]))
+}
+
+function saveWitnetJsonFiles(data) {
+    Object.entries(data).forEach(([key, obj]) => {
+        const filepath = `./witnet/${key}.json`
+        if (!fs.existsSync(filepath)) fs.writeFileSync(filepath, "{}");
+        const json = merge(JSON.parse(fs.readFileSync(filepath)), obj)
+        fs.writeFileSync(filepath, JSON.stringify(json, null, 4), { flag: "w+" })
+    })
 }
 
 function deleteExtraFlags(args) {
```

Comment:
I had to read this function to see if it is returning an array. Is this function working? `deleted` variable is not defined and should fail.